apps/prairielearn/src/pages/administratorWorkspaces/administratorWorkspaces.ts

```diff
@@ -0,0 +1,29 @@
+import asyncHandler = require('express-async-handler');
+import express = require('express');
+import { loadSqlEquiv, queryValidatedRows } from '@prairielearn/postgres';
+
+import { config } from '../../lib/config';
+import { AdministratorWorkspaces, WorkspaceHostRowSchema } from './administratorWorkspaces.html';
+
+const router = express.Router();
+const sql = loadSqlEquiv(__filename);
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const workspaceHostRows = await queryValidatedRows(
```

Comment:
Any reason why this is not using `queryRows` instead?

Reply:
I think this code predates the existence of `queryRows`? Either way, I changed this in 2171f0d.